src/amo/components/UserProfileEdit/index.js

```diff
@@ -317,6 +327,11 @@ export class UserProfileEditBase extends React.Component<Props, State> {
       }
 
       errorMessage = errorHandler.renderError();
+      window.scroll({
```

Comment:
I don't think we need this here.

Reply:
i added this here so that on deleting the profile picture, the user is redirected at top. So is this functionality not needed at all or is the piece of code is at wrong position?

Reply:
I added this so that if in username a person adds a wrong character then it redirects to top to display the error message- "Enter a valid username consisting of letters, numbers, underscores or hyphens."

Reply:
I added this so that if in username a person adds a wrong character then it redirects to top to display the error message- "Enter a valid username consisting of letters, numbers, underscores or hyphens."